### DISCUSSION TITLE
Icon colors on light theme

### DISCUSSION THREAD

**Discussion by ArioZarrin** [stefanobartoletti/nuxt-social-share#316] [Q&A]
my site has dark and light theme with Vuetify. 
I use nuxt-social-buttons. 

`   <SocialShare
          v-for="network in ['facebook', 'x', 'linkedin', 'email']"
          :key="network" :network="network" :styled="true" :label="false" />`
      
 Icons  have transparent background. the problem is that Icons in dark and light themes remain white. so they are not visible on light theme. 
 
 How to solve this problem?

**Comment by stefanobartoletti** [comment]
Default style of the component is meant to be overwritten by your own CSS rules, if you are using it
